app/src/main/java/org/standardnotes/notes/SettingsActivity.kt

```diff
@@ -35,6 +38,24 @@ class SettingsActivity : BaseActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        // TODO does this make sense here?
+        ExportUtil.clearExports(this)
```

Comment:
onDestroy() is very unpredictable, will happen immediately if you back out of the activity. Is that ok?